llvm/lib/CodeGen/MachineFunction.cpp

```diff
@@ -330,6 +325,18 @@ bool MachineFunction::shouldSplitStack() const {
   return getFunction().hasFnAttribute("split-stack");
 }
 
+Align MachineFunction::getPreferredAlignment() const {
+  Align PrefAlignment = Align(1);
```

Comment:
Instead of reassigning to the variable multiple times, it's better to reverse the order the following two conditions

Reply:
Done